src/mousetracker.js

```diff
@@ -166,7 +166,7 @@
         /**
          * The maximum distance allowed between two pointer click events
          * to be treated as a click gesture.
```

Comment:
This line should also be changed so `click` is `double-click`.

Reply:
done